esmf_regrid/esmf_regridder.py

```diff
@@ -57,7 +58,14 @@ def _weights_dict_to_sparse_array(weights, shape, index_offsets):
 class Regridder:
     """Regridder for directly interfacing with :mod:`esmpy`."""
 
-    def __init__(self, src, tgt, method="conservative", precomputed_weights=None):
+    class Method(Enum):
+        CONSERVATIVE = esmpy.RegridMethod.CONSERVE
+        BILINEAR = esmpy.RegridMethod.BILINEAR
+        NEAREST = esmpy.RegridMethod.NEAREST_STOD
+
+    NormType = Enum("NormType", ["FRACAREA", "DSTAREA"])
+
+    def __init__(self, src, tgt, method=None, precomputed_weights=None):
```

Comment:
I know `self.Method.CONSERVATIVE` doesn't work, but I don't get any linting errors when I try:

```suggestion
    def __init__(self, src, tgt, method=Method.CONSERVATIVE, precomputed_weights=None):
```